CHANGELOG.md

```diff
@@ -1,24 +1,118 @@
+# v0.5.0
+
+## Enhancements
+
+* Add API for fetching Fulcio configuration (https://github.com/sigstore/fulcio/pull/608)
+* Final challenge result removal 🎉  (https://github.com/sigstore/fulcio/pull/626)
+* Move email principal to package (https://github.com/sigstore/fulcio/pull/620)
+* Make prometheus port configurable (https://github.com/sigstore/fulcio/pull/625)
+* Move kubernetes principal to package (https://github.com/sigstore/fulcio/pull/619)
+* Move spiffe principal to package (https://github.com/sigstore/fulcio/pull/604)
+* Split pkg/server from pkg/api (https://github.com/sigstore/fulcio/pull/616)
+* Move github principal to its own package (https://github.com/sigstore/fulcio/pull/599)
+* Restict issuer claim mapping to email issuers (https://github.com/sigstore/fulcio/pull/606)
+* Move domain validation checks for URI/Username to service startup (https://github.com/sigstore/fulcio/pull/590)
+* Validate SPIFFE IDs and trust domains via library (https://github.com/sigstore/fulcio/pull/592)
+* Refactor challenge verification (https://github.com/sigstore/fulcio/pull/580)
+* identity: improve the documentation for Principal.Name() (https://github.com/sigstore/fulcio/pull/579)
+* challenges: remove ParseCSR (https://github.com/sigstore/fulcio/pull/578)
```

Comment:
Can we call this misc or maybe change the bug fix header to bug fix/misc? This was just switching to a sigstore/sigstore method.